### DISCUSSION TITLE
Is there any dev positions, or collaborator positions?

### DISCUSSION THREAD

**Discussion by Aka-but-nice** [3kh0/ext-remover#329] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Are there any dev positions, or collaborator positions?

**Comment by 3kh0** [comment, marked as answer]
The fact that you just joined is not gonna land you collab

**Reply by obkduishinijf** [reply]
how ironic that he got collab lmao

**Reply by Aka-but-nice** [reply]
its hard work, its seriously not just answer questions

**Reply by Aka-but-nice** [reply]
> The fact that you just joined is not gonna land you collab

@3kh0 _laughs at this misinformation_

**Reply by obkduishinijf** [reply]
> > The fact that you just joined is not gonna land you collab
> 
> @3kh0 _laughs at this misinformation_

real

**Comment by Aka-but-nice** [comment]
ik but gotta try

**Reply by 3kh0** [reply]
getting to the top of the most helpful leaderboard will boost that

**Reply by Aka-but-nice** [reply]
ok

**Reply by Aka-but-nice** [reply]
Will this boost my chances #347

**Reply by Aka-but-nice** [reply]
@3kh0 how do you get to the top of helpful leaderboard

**Reply by 3kh0** [reply]
answering questions

**Reply by Aka-but-nice** [reply]
ok

**Comment by Aka-but-nice** [comment]
@3kh0 Can you approve my pull request, my other one had many errors.
